client/src/components/Ticketing/ticketingmanager/Season/components/SeasonEvents.tsx

```diff
@@ -19,11 +19,12 @@ const SeasonEvents = (props: SeasonEventsProps) => {
   const handleGetAllEvents = async () => {
     const allEvents = await getAllEvents(token);
     if (allEvents) {
-      const {data} = allEvents;
-      const eventsInSeason = data.filter(
-        (event) => event.seasonid === seasonId,
+      const eventsInSeason = allEvents.filter(
+        (event) => event.seasonid_fk === seasonId,
+      );
+      const eventsNotInSeason = allEvents.filter(
```

Comment:
For clarity sake could we rename this to `eventsNotInASeason`. At the moment, it looks like it means all events that are not in this season

Reply:
That makes sense, will do!

Reply:
Now that I took a look, it actually does refer to all events not in the current season, so the naming scheme makes sense. Since I store all the events in `eventsInSeason` variable, everything else is not in the season

Reply:
But isn't it checking for all events that have a `null` foreign key. That means that events that are in _different_ seasons are also not included in this var

Reply:
Hmm yeah you're right in that sense. I guess I just grouped all the events that were not in the current season (whether it was null or a part of another season) together. I'll changed that to be more clear!

Reply:
I guess if it makes more sense you could do something like: `eventsNotInAnySeason`